elements/pfe-tabs/src/pfe-tab.scss

```diff
@@ -264,7 +265,8 @@ $LOCAL-VARIABLES: map-deep-merge($LOCAL-VARIABLES, $backwards-compatibility);
 
 :host([on="dark"][pfe-variant="earth"][aria-selected="true"]) {
   --pfe-tabs--Color--focus: #{pfe-var(text--on-dark)};
-  background-color: pfe-var(surface--darkest);
+  --pfe-tabs--BackgroundColor: #{pfe-var(surface--darkest)};
```

Comment:
```suggestion
  --pfe-tabs--BackgroundColor: #{pfe-var(surface--darkest)};
  --pfe-tabs--theme: #{pfe-var(surface--darkest--theme)};
```

Reply:
will this have an impact since this is the individual tab, so there wont be any rogue content here which could have the wrong text color?

Reply:
Ah, probably not #dust 🌫